<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintenanceToolsWindow</class>
 <widget class="QDialog" name="MaintenanceToolsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maintenance tools</string>
  </property>
  <widget class="QGroupBox" name="groupBox_syringe_dimensions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Syringe dimensions</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_syringe">
     <item row="0" column="0">
      <widget class="QLabel" name="label_syringe_volume">
       <property name="text">
        <string>Syringe volume (µL):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_syringe_volume">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_syringe_diameter">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_syringe_diameter">
       <property name="text">
        <string>Syringe diameter (mm):</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_syringe_calibration">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Syringe calibration</string>
   </property>
   <widget class="QToolButton" name="button_calibrate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>70</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_max_pumping_volume">
       <property name="text">
        <string>Max. pumping vol. (µL):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_max_pumping_volume">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_syringe_purge">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Syringe purge</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>30</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_purge_cycles">
       <property name="text">
        <string>Number of cycles:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_purge_cycles"/>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="button_purge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Purge</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
